packages/dbos-cloud/cli.ts

```diff
@@ -11,7 +11,7 @@ import { Command } from 'commander';
 import { login } from "./users/login.js";
 import { registerUser } from "./users/register.js";
 import { createUserDb, getUserDb, deleteUserDb, listUserDB, resetDBCredentials, linkUserDB, unlinkUserDB, restoreUserDB } from "./databases/databases.js";
-import { launchDashboard, getDashboardURL } from "./dashboards/dashboards.js";
+import { launchDashboard, getDashboardURL, deleteDashboard } from "./dashboards/dashboards.js";
```

Comment:
If I understand it correctly, we can mark `launchDashboard` as deprecated because on the cloud side `getDashboardURL` will create one if it doesn't exist?

Reply:
Yes we can! I wasn't 100% sure on how best to do it? Something like console.warn?

Reply:
See https://github.com/dbos-inc/dbos-transact/pull/476/commits/169df16484df0fe7c1bc79ab2703a87a7b41f453#diff-b28f7a495d363d965d0679918e0445e05040b4ab23cc4fb6f99c5acb538c7a2dR10